SecretSanta/SecretSanta.Business.Tests/UserRepositoryTests.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SecretSanta.Business;
+using SecretSanta.Data;
+
+namespace SecretSanta.Business.Tests
+{
+    [TestClass]
+    public class UserRepositoryTests
+    {
+        Mock<User> mockValidUser = new Mock<User>();
+        Mock<User> mockEditedValidUser = new Mock<User>();
+        Mock<User> mockEditedInvalidUser = new Mock<User>();
```

Comment:
For fields we specify access modifiers.
```suggestion
        private Mock<User> mockValidUser = new Mock<User>();
        private Mock<User> mockEditedValidUser = new Mock<User>();
        private Mock<User> mockEditedInvalidUser = new Mock<User>();
```